cypress/integration/listingFlows/deleteListing.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Delete listing', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.viewport('macbook-16');
+
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user).then(() => {
+        cy.createListing({
+          title: 'Test Listing',
+          content: `This is a test listing's contents.`,
+        }).then(() => {
+          cy.visit('/listings/dashboard');
+        });
+      });
+    });
+  });
+
+  it('deletes a listing', () => {
```

Comment:
I think it might be worthwhile starting this test by asserting that "Test Listing" appears in the dashboard. Then as the final step on line 28 we'd be able to assert that "Test Listing" no longer appears, instead of relying on a class selector / empty HTML which can be a lot more changeable